### ISSUE TITLE
start unicorn_rails (instead of unicorn)

### ISSUE THREAD

**Issue by ronald** [sosedoff/capistrano-unicorn#6]
Thanks for providing this. 

It works great for Rack-Apps. 

Unfortunately I've to deploy a Rails 2.3-App using unicorn_rails but I can not configure the unicorn binary.

Would be nice if we can swap
bundle exec unicorn 
by 
bundle exec unicorn_rails
or 
bin/unicorn_rails
or
bin/unicorn

(for using rbenv / binstubs...
bundle -v
Bundler version 1.1.rc.7
bundle install --binstubs --shebang ruby-local-exec)

**Comment by sosedoff** [comment]
What is the problem exactly? Also, what version are you using ?

Im also running capistrano-unicorn on rails version 3.0, 3.1 and 3.2. and it seems to be fine

**Comment by ronald** [comment]
capistrano-unicorn-0.1.5 + unicorn 4.2 + Rails 2.3.11

Rails apps older than 3.0 don't support Rack and can not be started using "unicorn" (Rack-Server).
Old Rails apps are supported by using "unicorn_rails".

**\* [Unicorn] Starting...
- executing "cd /app/current && BUNDLE_GEMFILE=/app/current/Gemfile bundle exec unicorn -c /app/current/config/unicorn/production.rb -E production -D"
  servers: ["app-server"]
  [app-server] executing command
  [app-server] sh -c 'cd /app/current && BUNDLE_GEMFILE=/app/current/Gemfile bundle exec unicorn -c /app/current/config/unicorn/beta-production.rb -E production -D'
  **\* [err :: app-server] /app/shared/bundle/ruby/1.8/gems/unicorn-4.2.0/lib/unicorn/configurator.rb:610:in `parse_rackup_file': rackup file (config.ru) not readable (ArgumentError)
  *** [err :: app-server] from /app/shared/bundle/ruby/1.8/gems/unicorn-4.2.0/lib/unicorn/configurator.rb:76:in`reload'
  [..]
  **\* [err :: app-server] from /app/shared/bundle/ruby/1.8/bin/unicorn:19
  **\* [err :: app-server] master failed to start, check stderr log for details
  **\* [err :: app-server]

**Comment by sosedoff** [comment]
Oh, i dont think i tested it for Rails 2 apps. Might take a look later.

Could you provide a config.ru contents? Seems like you have an error there.

**Comment by ronald** [comment]
no rack = no config.ru

**Comment by ronald** [comment]
Can you please provide an option to set the unicorn executable that defaults to "bundle exec unicorn"?

Rails 2.x users could change this to "bundle execute unicorn_rails".

**Comment by sosedoff** [comment]
Yeah, i can setup a execution mode for unicorn. 

In config file it should be something like:

``` ruby
set :unicorn_binary, 'unicorn_rails'
```

I'll make an update today.

**Comment by sosedoff** [comment]
Released 0.1.6. 

Now you can setup your unicorn executable file:

```
set :unicorn_bin, "unicorn_rails"
```

It is set to be "unicorn" by default.

**Comment by ronald** [comment]
works great. thanks a lot.
